rancher2/structure_cluster_rke_config_test.go

```diff
@@ -57,6 +57,7 @@ func testClusterRKEConfig() {
 			"network":               testClusterRKEConfigNetworkInterfaceCanal,
```

Comment:
`WindowsPrefixPath` field should also be added to `testClusterRKEConfigConf` struct

```
...
	testClusterRKEConfigConf = &managementClient.RancherKubernetesEngineConfig{
...
		PrefixPath:          "terraform-test",
+		WindowsPrefixPath:   "c:\\terraform-test",
		PrivateRegistries:   testClusterRKEConfigPrivateRegistriesConf,
...
```